jumpcloud-ADMU/Powershell/ProgressForm.ps1

```diff
@@ -0,0 +1,363 @@
+
+The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham.</TextBlock>
+                        </ScrollViewer>
+                    </Border>
+                </Expander>
+            </StackPanel>
+
+            <Grid Grid.Row="2" Margin="0,20,10,0" HorizontalAlignment="Right" VerticalAlignment="Top" Height="26" Width="240" >
+
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="Auto"/>
+                    <ColumnDefinition Width="0"/>
+                    <ColumnDefinition Width="Auto"/>
+                    <ColumnDefinition Width="0"/>
+                    <ColumnDefinition Width="Auto"/>
+                </Grid.ColumnDefinitions>
+                <Button Grid.Column="0" Name="ViewLogButton" Content="View Log" Height="26" Width="70"  IsEnabled="False"  Margin="0,0,10,0" HorizontalAlignment="Center">
+                    <Button.Resources>
+                        <Style TargetType="{x:Type Border}">
+                            <Setter Property="CornerRadius" Value="3"/>
+                        </Style>
+                    </Button.Resources>
+                </Button>
+                <Button Grid.Column="2" Name="StartJCADMUButton" Content="Rerun"  Height="26" Width="70" IsEnabled="False" Margin="0,0,10,0">
+                    <Button.Resources>
+                        <Style TargetType="{x:Type Border}">
+                            <Setter Property="CornerRadius" Value="3"/>
+                        </Style>
+                    </Button.Resources>
+                </Button>
+                <Button Grid.Column="4" Name="ExitButton" Content="Exit" Height="26" Width="70"  VerticalAlignment="Top" IsEnabled="False" Margin="0,0,10,0">
+                    <Button.Resources>
+                        <Style TargetType="{x:Type Border}">
+                            <Setter Property="CornerRadius" Value="3"/>
+                        </Style>
+                    </Button.Resources>
+                </Button>
+            </Grid>
+        </Grid>
+    </Grid>
+</Window>
```

Comment:
![Screenshot 2024-07-12 at 1 38 14 PM](https://github.com/user-attachments/assets/a9e24388-bc8e-4488-91be-9df835952e03)

Similar items to discuss with Progress Form:

- The logo looks aligned correctly here
- The "Migration Status", "Log" boxes & buttons should just be right aligned to the right border.
- The Log box isn't left aligned to the boarder
- The margin between the log box and the buttons looks a bit larger than the margin between "Migration Details" and "Migration Status" these should be the same.

Reply:
![Screenshot 2024-07-12 at 1 59 19 PM](https://github.com/user-attachments/assets/46694a8c-5bf7-4d0b-be80-32f4e4ae2d11)